tool/src/graal/resource-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
How is this used?

Reply:
this is fed into the graal build, you can see it being used in the drone config